helm/hpcc/values.yaml

```diff
@@ -95,6 +95,7 @@ storage:
   #   includeDeviceInPath: false            # Is the device number appended to the mount for the physical path for a file?  (Only required in unusual situations)
   #   hosts: <name>                         # Name of the host group for bare metal - must match the name of the storage plane..
   #   secret: <secret-id>                   # what secret is required to access the files.  This could optionally become a list if required (or add secrets:).
+  #   defaultSprayParts: 4                  # the partitions created when spraying (if not explicity specified)
```

Comment:
what happens if not set, and the plane is spray to, without explicitly saying how many parts?
Does it fallback to some global default somewhere, or it an error?

Reply:
If this is not set, it defaults to '1'.  I'll put a comment to explain this.